_posts/en/newsletters/2021-07-07-newsletter.md

```diff
@@ -0,0 +1,144 @@
+asked in IRC why, but unless there's a *really* compelling reason, I'd
+prefer to capitalize.  I won't die on this hill, but I'm willing to lose
+a little blood to prevent terms like iPhone that are super annoying to use
+at the beginning of a sentence. -->
+
+- **BLIPs:** Ryan Gentry [posted][gentry blips] to the Lightning-Dev
+  mailing list a proposal for a collection of Bitcoin Lightning
+  Improvement Proposals (BLIPs), documents that describe extensions and
+  applications of LN that will benefit from interoperability standards.
+  René Pickhardt linked to an almost [identical proposal][pickhardt
+  lips] he made in 2018.
+
+    In discussion, the idea seemed to have broad support, although
+    [concerns][teinturier blips] were raised that it doesn't actually
+    solve the barrier to getting those standards incorporated in the
+    base BOLTs documents---that barrier being experienced developers
+    lacking enough time to review the many community proposals.  If
+    BLIPs are merged without significant review, that increases the
+    chance that they'll contain bugs or that they'll fail to achieve
+    broad support from multiple stakeholders, leading to fragmentation
+    as different projects adopt competing standards.  Still,
+    non-mainline protocols are already being created and most discussion
+    participants seemed to believe providing a well-known archive where
+    documentation about those protocols could be published would be
+    primarily beneficial.
+
+- **Zero-conf channel opens:** Rusty Russell started a
+  [discussion][russell zeroconf] on the Lightning-Dev mailing list about
+  standardizing the handling of zero-conf channels, also known under the
+  name *turbo channels*.  These are new single-funded channels where the
+  funder gives some or all of their initial funds to the acceptor.
+  Those funds are not secure until the channel open transaction receives
+  a sufficient number of confirmations, so there's no risk to the
+  acceptor spending some of those funds back through the funder using
+  the standard LN protocol.
+
+    For example, Alice has several BTC in an account at Bob's custodial
+    exchange.  Alice asks Bob to open a new channel paying her 1.0 BTC.
+    Because Bob trusts himself not to double-spend the channel he just
+    opened, he can allow Alice to send 0.1 BTC through his node to
+    third-party Carol even before the channel open transaction has
+    received a single confirmation.
+
+    {:.center}
+    ![Zero-conf channel illustration](/img/posts/2021-07-zeroconf-channels.png)
+
+    Some LN implementations already support the idea in a
+    non-standardized way and all discussion participants seemed to favor
+    standardizing it.  The exact details to use were still being
+    discussed at the time of writing.
+
+## Preparing for taproot #3: taproot descriptors
+
+*A weekly [series][series preparing for taproot] about how developers
```

Comment:
We should also update the introductory text here: https://bitcoinops.org/en/newsletters/2021/06/23/#preparing-for-taproot-1-bech32m-sending-support and here: https://bitcoinops.org/en/newsletters/2021/06/30/#preparing-for-taproot-2-is-taproot-even-worth-it-for-single-sig to point to the series blog post.